lib/src/layer/polygon_layer.dart

```diff
@@ -84,27 +94,29 @@ class PolygonLayer extends StatelessWidget {
     final origin = map.pixelOrigin;
     final offset = Offset(origin.x.toDouble(), origin.y.toDouble());
 
-    final Iterable<Polygon> pgons = polygonCulling
+    final List<Polygon> pgons = polygonCulling
```

Comment:
Is there a reason why you converted the `Iterable<Polygon>` to a `List<Polygon>` I don't really see the benefits here 🤔 
Otherwise everything else looks good to me.

(Do you want to double check @JaffaKetchup?)

Reply:
Since I recently changed it from an List to an Iterable I felt like going full circle :). Jokes aside, I added the length comparison as quick true-positive check. Since both, calculating the length and the hashing, require a full traversal (e.g. when the raster cache is being hit), I felt the benefit of delaying the list conversion was harder to justify. Hope that makes sense. In other words, I can change it back and it might be marginally letss work in cases where the length actually differs and marginally more work in all others.